### PR TITLE
metricbeat 6.0.0

### PR DESCRIPTION
**Pull request by jasontedor**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ilovezfs — CHANGES_REQUESTED** [review]
This should use a resource block and `setup_install_args` instead of the virtualenv DSL.

**Comment by ilovezfs** [comment]
virtualenv doesn't seem to be used by the build.

File: Formula/metricbeat.rb [hunk 1]
```diff
@@ -72,16 +79,16 @@ def plist; <<~EOS
...
-    metricbeat_pid = fork do
-      exec bin/"metricbeat", "-c", testpath/"metricbeat.yml",
-      "--path.data=#{testpath}/data", "--path.logs=#{testpath}/logs"
+    pid = fork do
+      exec bin/"metricbeat", "-path.config", testpath/"config", "-path.data", testpath/"data"
     end
 
     begin
-      sleep 2
+      sleep 30
```
Comment (ilovezfs) [inline Formula/metricbeat.rb]: why so long?

Comment (jasontedor) [reply Formula/metricbeat.rb]: Because the changes don’t get flushed to disk fast enough so the test ends up being flaky with a short sleep.

**Comment by jasontedor** [comment]
`virtualenv` is used in `make kibana` call.

**Comment by ilovezfs** [comment]
It doesn't seem to be used. I commented it out and build was still successful.

**Comment by jasontedor** [comment]
Okay, I’ll work on removing it. Thanks for checking.

**Comment by jasontedor** [comment]
I pushed .

**Comment by ilovezfs** [comment]
Merged with all its 30 seconds of :zzz: :sleeping: glory, @jasontedor!

**Comment by jasontedor** [comment]
Thanks @ilovezfs.

**Pull request closed without merging**